wordpress-mobile/WordPress-iOS
# Dark mode: NUX improvements

Refs #12320. This PR implements some improvements to the appearance of the login flow for dark mode.

* Updated primary and secondary button styles
* Fixed invisible text in text fields, and increased padding between icon and text
* Selectively show dark or light Sign in with Apple button based on user interface style

![dark-before-after](https://user-images.githubusercontent.com/4780/63635486-2cf3a780-c65b-11e9-9a10-87a371f0d60d.png)

Shouldn't be noticeable changes in light mode.

**To test:**

* Build and run in iOS 13, light and dark mode, and in iOS 12
* Check things look as expected

Update release notes:


cc @mattmiklic
